patches/tModLoader/Terraria/ID/NPCID.cs.patch

```diff
@@ -17,3 +17,228 @@
  			public static Dictionary<int, int> SpecialSpawningRules = new Dictionary<int, int> {
  				{ 259, 0 },
  				{ 260, 0 },
+@@ -4125,7 +_,7 @@
+ 
+ 				dictionary.Add(6, value);
+ 				value = new NPCBestiaryDrawModifiers(0) {
+-					CustomTexturePath = "Images/UI/Bestiary/NPCs/NPC_7",
++					CustomTexturePath = "Terraria/UI/Bestiary/NPCs/NPC_7", // TODO: This and all subsequent filepath patches are temporary until we overhaul the asset system. Very hacky.
+ 					Position = new Vector2(20f, 29f),
+ 					PortraitPositionXOverride = 0f,
+ 					PortraitPositionYOverride = 10f
+@@ -4143,7 +_,7 @@
+ 
+ 				dictionary.Add(9, value);
+ 				value = new NPCBestiaryDrawModifiers(0) {
+-					CustomTexturePath = "Images/UI/Bestiary/NPCs/NPC_10",
++					CustomTexturePath = "Terraria/UI/Bestiary/NPCs/NPC_10",
+ 					Position = new Vector2(2f, 24f),
```

Comment:
There have to be smarter solutions.

Reply:
Easiest way without causing file path issues for mods. This was never meant to be permanent.

Reply:
Reduced the amount of patches to 1, and also removed the need to remove 'Images' prefix.